internal/command/views/test.go

```diff
@@ -220,17 +221,22 @@ func (t *TestHuman) DestroySummary(diags tfdiags.Diagnostics, run *moduletest.Ru
 	if diags.HasErrors() {
 		t.view.streams.Eprint(format.WordWrap(fmt.Sprintf("OpenTofu encountered an error destroying resources created while executing %s.\n", identifier), t.view.errorColumns()))
 	}
+
 	t.Diagnostics(run, file, diags)
 
 	if state.HasManagedResourceInstanceObjects() {
-		t.view.streams.Eprint(format.WordWrap(fmt.Sprintf("\nOpenTofu left the following resources in state after executing %s, and they need to be cleaned up manually:\n", identifier), t.view.errorColumns()))
+		t.view.streams.Eprint(format.WordWrap(fmt.Sprintf("\nOpenTofu left the following resources in state after executing %s, the left-over resources can be cleaned using the statefile written to disk(errored_test.tfstate) or can be done manually:\n", identifier), t.view.errorColumns()))
```

Comment:
I think the wording of this is a bit unclear, making me end users should be running some commands against the errored_test.tfstate file to remove things.

When in reality this feature is to just dump the statefile to disk so that people have a reference. Or am I incorrect in this assumption?

Reply:
https://github.com/opentofu/opentofu/issues/248#issuecomment-1837497775

Here is a reference to what was discussed in the issue, I assumed that if we stored a state file then there was a way by which we could source that particular state file and then run `tofu destroy` or other tofu commands. Correct me if I am wrong please.

Reply:
I think in this case the user needs to clean up by hand, I wouldn't encourage them to run commands against something that's broken, there's a chance to make an even bigger mess.

Reply:
I agree with @janosdebugs. Let's keep it simple. If a need arises, we can add this as a separate feature.